library/src/main/java/com/novoda/downloadmanager/VersionOneToVersionTwoMigrator.java

```diff
@@ -135,7 +139,11 @@ private void deleteFrom(SqlDatabaseWrapper database, List<Migration> migrations)
         for (Migration migration : migrations) {
             Batch batch = migration.batch();
             database.rawQuery(DELETE_BY_ID_QUERY, batch.getDownloadBatchId().stringValue());
+            for (Migration.FileMetadata metadata : migration.getFileMetadata()) {
+                // TODO: Handle error cases, pay attention to file.delete() result
```

Comment:
Would be better represented as a ticket in GH `Issues` with a link here to the ticket. This way you are forced to provide more information than just `Handle error cases` which someone new on the project would not understand.